onnxruntime/python/tools/transformers/fusion_skiplayernorm.py

```diff
@@ -176,11 +173,11 @@ def fuse(self, node, input_name_to_nodes, output_name_to_node):
 
         self.nodes_to_remove.extend(subgraph_nodes)
         inputs = [
-            node.input[1 - add_input_index],
-            matmul.output[0],
+            sln_input,
```

Comment:
Here we put matmul output to the first input, and skip to the second according to operator spec. So we need update some test data accordingly.